dev-python/pydotplus/pydotplus-2.0.2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
sure, but upstreams are listing wildest things there. please test with 3.3 and 3.5 as well.